docs/ansible/roles/debops.postldap/guides.rst

```diff
@@ -28,7 +28,99 @@ email addresses. The default configuration uses first the aliases set by
+See `secret__ldap_fqdn`.
+
+.. code-block:: yaml
+  :caption: ansible/inventory/group_vars/hetzner/ldap.yml
+
+  ---
+
+  # Enable LDAP, as is deactivated by default
+  ldap__enabled: True
+
+  ldap__domain: 'mydomain.net'
+
+  # Domain used for LDAP base DN and to select default LDAP server.
+  secret__ldap_domain: 'mydomain.net'
+
+  # LDAP server hostname / internal IP address which holds the database.
+  # ldap_* modules will connect to it natively, so it should be available at least from the Ansible Controller.
+  secret__ldap_fqdn: 'ldap.hetzner.mydomain.net'
+
+.. code-block:: yaml
+  :caption: ansible/inventory/host_vars/skynet.mydomain.net/slapd.yml
+
+  ---
+
+  ## Network access to OpenLDAP server
+  # Firewall Settings
+  #   Block connections to the OpenLDAP via system firewall and TCP Wrappers from any host (aka Internet);
+  #   Hosts that can connect must be specified via the slapd__*_allow variables.
+  slapd__accept_any: false
+
+  slapd__group_allow:
+    # IPv4/v6 localhost
+    - '127.0.0.1'
+    - '::1'
+    - '{{ ansible_default_ipv4.address
+          if (ansible_default_ipv4|d() and ansible_default_ipv4.address|d() )
+          else "127.0.0.1" }}'
+    - '{{ ansible_default_ipv6.address
+          if (ansible_default_ipv6|d() and ansible_default_ipv6.address|d() )
+          else "::1" }}'
+
+    # Hetzner internal network
+    - '10.10.10.0/28'
+
+.. code-block:: yaml
+  :caption: ansible/inventory/host_vars/mail-server.mydomain.net/pki.yml
+
+  ---
+
+  ### Create TLS Certs for the mail server
+  #
+  # In order to sign the cert by Let's Encrypt CA install nginx in the 'mail-server',
+  # so that the acme script can work.
+  pki_host_realms:
+    - name: 'mail.mydomain.net'
+      acme: false
+      domains:
+        - 'mail.mydomain.net'
+        - 'smtp.mydomain.net'
+        - 'imap.mydomain.net'
+        - 'mail-server.mydomain.net'
+
 .. code-block:: yaml
+  :caption: ansible/inventory/host_vars/mail-server.mydomain.net/postfix.yml
+
+  ---
+
+  # basic Postfix SMTP server with configuration similar to the "Internet Site"
+  # MTP service listens for connections on port 25 from all hosts.
+  # Mail relay is authorized from localhost, other hosts are deferred.
+
+  postfix__domain: 'mydomain.net'
+
+  postfix__pki_realm: 'mail.mydomain.net'
+
+  # Enable debugging
+  postldap__no_log: False
+
+  postldap__ldap_private_subtree: True
 
-  Work in progress...
+  # Use a different LDAP filter to look up user accounts in the directory.
+  postldap__ldap_user_filter: '(&
+                                 (objectClass=inetOrgPerson)
+                                 (objectClass=postfixUser)
+                               )
+                               (mailEnabled=TRUE)
+                               (|
+                                 (authorizedService=postfix)
+                                 (authorizedService=mail)
+                                 (authorizedService=\*)
+                               )'
```

Comment:
This included in the example begs the question why this filter wasn't included by default in the role... Check my other comment about the user filter - I think that with the improved default this particular example could be removed from the guide.